### ISSUE TITLE
[dropdown] dropdown hides input icons

### ISSUE THREAD

**Issue by azmeuk** [fomantic/Fomantic-UI#2172]
# Bug Report
When using a dropdown to display a `<label>` (in JS with `$("select").dropdown()`), input with an icon, the icon is not visible.

## Steps to reproduce
```html
<div class="ui fluid left icon input">
  <i class="asterisk icon"></i>
  <div class="ui fluid dropdown multiple selection"><i class="dropdown icon"></i>
   ...
  </div>
</div>
```
## Expected result
I would expect to see the icon, and that the content of the dropdown to be displayed just next to the icon.

## Actual result
The icon is not visible, and when an item has been selected in the dropdown, it is displayed where the icon should be.

## Testcase
https://jsfiddle.net/azmeuk/dr97ubh2/16/

## Screenshot
![Screenshot 2021-12-07 at 15-01-27 Fomantic-UI Sandbox - JSFiddle - Code Playground](https://user-images.githubusercontent.com/60163/145042960-af2f746a-df03-4fd7-b7fd-fea03c1a2393.png)

## Version
2.8.8

**Comment by lubber-de** [comment]
You have to put the icon _after_ the dropdown node instead of before.
See https://jsfiddle.net/lubber/tk4Lf9s6/4/

However the icon isn't properly positioned when used in a dropdown which is fixed by #2183

**Comment by azmeuk** [comment]
Thank you!
